tests/actiontest.cpp

```diff
@@ -169,9 +152,9 @@ private slots:
         QCOMPARE(a1->isChecked(), false);
         QCOMPARE(changeSpy.size(), 2);
     }
-
 };
 
 QTEST_MAIN(ActionTest)
 
+#include "moc_baseprobetest.cpp"
```

Comment:
I very much like the idea. Can we compile this only once like we do for the probe object, and add this via CMake though, especially since there is now the gammaray_add_probe_test macro? Would avoid the extra moc include too.